EZP-31784 - Autosave description (#73)
ezsystems/user-documentation: docs/publishing/publishing.md

Note that you cannot edit an unpublished draft created by another user.

### Autosave

> eZ Platform comes with an Autosave feature that helps you preserve your work in an event of a failure while you edit a content item or product description.

Reviewer: What do you think about:
```suggestion
You work is saved automatically. Autosave helps you preserve your progress in an event of a failure while you edit a Content item or product.
```
This way avoids mentioning a feature, in the spirit of "don't describe features, describe what the user can do".
Although the passive could be fixed further...

> To recover your work, open the most recent draft in the **Versions** tab of the content item or
> product description.

Reviewer: ```suggestion
To recover your work, open the most recent draft in the **Versions** tab of the Content item.
```
If product is stored in the content tree, it is a Content item, no need to mention it explicitly.

Alternatively, open the most recent draft of your work on the **My dashboard** page, the **Drafts** table.

> The feature is enabled by default, and set to save a draft of your work every 60 seconds.

Reviewer: ```suggestion
Autosave is enabled by default, and set to save a draft of your work every 60 seconds.
```

> You can toggle Autosave or change the time between saving attempts in **User settings**, by changing

Reviewer: ```suggestion
You can toggle autosave or change the time between saving attempts in **User settings**, by changing
```
